### PR TITLE
Update items displayed in help dialog

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This pull request updates the items displayed in the help dialog.

### CODE REVIEW

File: asreview/webapp/help.json [hunk 1]
```diff
@@ -5,24 +5,24 @@
       "title": "Frequently asked questions",
       "items": [
         {
-          "title": "Has the use of ASReview in systematic reviews been validated?",
-          "url": "https://github.com/asreview/asreview/discussions/556"
+          "title": "What is the idea behind active learning?",
```
Comment (terrymyc) [inline asreview/webapp/help.json:8]: This question sounds too general. How about "How does active learning help with systematic reviews?"

File: asreview/webapp/help.json [hunk 2]
```diff
@@ -5,24 +5,24 @@
...
         {
           "title": "How to deal with records that do not have abstracts?",
-          "url": "https://github.com/asreview/asreview/discussions/553"
+          "url": "https://asreview.nl/blog/the-importance-of-abstracts/"
         },
         {
-          "title": "How can I add more publications while I have already started screening in ASReview?",
-          "url": "https://github.com/asreview/asreview/discussions/562"
+          "title": "How to contribute to the Open Source Project?",
+          "url": "https://github.com/asreview/asreview/blob/master/CONTRIBUTING.md"
```
Comment (terrymyc) [inline asreview/webapp/help.json:25]: This question and answer are irrelevant to most of our users. They don't have a programming background and won't be able to ask or read the answer to this question.

**Review by terrymyc — CHANGES_REQUESTED** [review]

File: asreview/webapp/help.json (hunk 1 above)
Comment (J535D165) [reply asreview/webapp/help.json]: Thanks. Good suggestion.

File: asreview/webapp/help.json [hunk 3]
```diff
@@ -5,24 +5,24 @@
...
         },
         {
           "title": "How to deal with records that do not have abstracts?",
-          "url": "https://github.com/asreview/asreview/discussions/553"
+          "url": "https://asreview.nl/blog/the-importance-of-abstracts/"
         },
         {
-          "title": "How can I add more publications while I have already started screening in ASReview?",
-          "url": "https://github.com/asreview/asreview/discussions/562"
+          "title": "How to contribute to the Open Source Project?",
```
Comment (J535D165) [inline asreview/webapp/help.json:24]:
```suggestion
          "title": "How to contribute to the open source ASReview project?",
```

**Pull request merged**